## BEEM011 Exercise 6: Week 8

### Single and Joint Hypothesis Tests

Amy Binner and Eva Poen

Answers

## Useful R code

CRITICAL VALUES

significance level = alpha

#### T-distribution

<code> t_crit = qt(alpha, df) </code> 

#### Normal-distribution 

<code> z_crit = qnorm(alpha) </code> 

#### F-distribution 

<code> f_crit = qf(alpha, df1, df2, lower.tail = FALSE) </code> 

#### Testing joint / multiple hypothesis tests

$ \hat{Y_i} = \hat{\beta_0} + \hat{\beta_1} X_{1i} + \hat{\beta_2} X_{2i} $

model <- lm(Y~X1+X2, data = mydata)
conditions = c("X1", "X2")

<code> linearHypothesis(model, conditions) </code>

# Question 1

The Solow growth model suggests that countries with identical saving rates and population growth rates should converge to the same per capita income level. This result has been extended to include investment in human capital (education) as well as investment in physical capital. This hypothesis is referred to as the "conditional convergence hypothesis," since the convergence is dependent on countries obtaining the same values in the driving variables. 

To test the hypothesis, you collect data from the Penn World Tables on the average annual growth rate of GDP per worker (g6090) for the 1960-1990 sample period, and regress it on the (i) initial starting level of GDP per worker relative to the United States in 1960 (RelProd60), (ii) average population growth rate of the country ($g_n$), (iii) average investment share of GDP from 1960 to 1990 (SK - remember investment equals savings), and (iv) educational attainment in years for 1985 (Educ). 

The results for 100 countries is as follows (numbers in parentheses are for *heteroskedasticity-robust* standard errors):

$ \hat{g_{6090}} = 0.004 - 0.172 \times g_n + 0.133 \times S_K + 0.002 \times Educ - 0.044 \times RelProd_{60} $
 
   &emsp; &emsp; &emsp;(0.007) &emsp; (0.209)  &emsp; &emsp;(0.015)    &emsp;&emsp;&emsp;(0.001)    &emsp;&emsp; &emsp;&emsp;(0.008)
   
   
$R^2=0.537, SER = 0.011$

## 1 a)

Is the coefficient on $RelProd_{60}$ significantly different from zero at the 5% level? At the 1% level? Use a two sided alternative.


Do this using the numbers, do not try to load the data.

Recall:

$$ t = {\hat{\beta} \over SE(\hat{\beta}) }$$

In [42]:
## Edit the code below 
# Header: Exercise 6 
# Author: 
# Date:
# Candidate number:


# Q1 (a) calculate t-stat, p-value and critical values
# Calculate the t-statistic
t.relprod <- -0.044/0.008

# Find the associated p-value
p.relprod <- 2*pnorm(-abs(t.relprod))
cat("p-value:",p.relprod)

# Calculate the critical values for a two sided test at the 5% and 1% significance level
z.2sided.1 <- qnorm(0.01/2)
z.2sided.5 <- qnorm(0.05/2)

cat("\n1% lower critical value: ", z.2sided.1)
cat("\n5% lower critical value: ", z.2sided.5)

# Decide whether or not to reject the null hypothesis
# H_0: beta_relprod = 0
# H_1: beta_relprod \ne 0

cat("\nReject null hypothesis at 5%:", p.relprod < 0.05)
cat("\nReject null hypothesis at 1 %:", p.relprod < 0.01)

p-value: 3.797912e-08
1% lower critical value:  -2.575829
5% lower critical value:  -1.959964
Reject null hypothesis at 5%: TRUE
Reject null hypothesis at 1 %: TRUE

## 1 b i)	

Test for the significance of the other slope coefficients. 

Should you use a one-sided alternative hypothesis or a two-sided test? 

Will the decision for one or the other influence the decision about the significance of the parameters? 

Should you always eliminate variables which carry insignificant coefficients?

In [23]:
# t g_n
t.gn <- -0.172 / 0.209
t.sk <- 0.133 / 0.015
t.educ <- 0.002 / 0.001

cat("\nt.gn =", round(t.gn,2), "\nt.sk =", round(t.sk,2),"\nt.educ =", round(t.educ,2))


t.gn = -0.82 
t.sk = 8.87 
t.educ = 2

#### Enter your answers here 

The t-statistics are –0.82. 8.87, and 2.00. Hence the coefficient on population growth is not statistically significant.  You can use a one-sided alternative hypothesis test since economic theory gives you information about the expected sign on these variables. 

In the above case, however, the decision will not be influenced by the choice of a one-sided or two-sided test, an insignificant variable should not be eliminated based on the significance test. Instead, it should be left in the equation, but the high p-value (or the low test statistic) should be flagged to the reader, and the reader should decide themselves how convincing the evidence is in favor of the theory. The variable may be controlling for other factors.

# Question 2

Using the 420 observations of the California School data set, you estimate the following relationship:

$$\hat{TestScore} = 681.44 - 0.61LchPct $$

$$n=420, R^2=0.75, SER=9.45 $$

where TestScore is the test score and LchPct is the percent of students eligible for subsidized lunch (average = 44.7, max = 100, min = 0).

# 2 a)

Interpret the regression result

#### Enter your answer here

For every 1 percentage point increase in students eligible for subsidized lunch, average test scores are expected to go down by 0.61 points. If a school has no students eligible for subsidized lunch, then the average test score is approximately 681 points. 75% of the variation in test scores is explained by our model. On the average, the true average test scores deviate from the predicted average test scores  using the above regression model by 9.45 points. 

# 2 b)

In your interpretation of the slope coefficient in (a) above, does it matter if you start your explanation with "for every x percent increase" rather than "for every x percentage point increase"?

#### Enter your answer here

Since your RHS variable is measured already in percent (%), it makes sense to increase that variable by 10 percentage points (say), rather than by 10 percent. If LchPct increases from 20 to 30, then this represents an increase of 10 percentage points, or an increase of 50 percent.

# 2 c)

The "overall" regression F-statistic is 1149.57. What are the degrees of freedom for this statistic?

F(df1, df2)

#### Enter your answer here

q = 1, k = 1, n = 420

There is  1 degree of freedom in the numerator, and 418 (∞) degrees of freedom in the denominator, 420 - 1 - 1. 

# 2 d)

Find the critical value of the F-statistic at the 1% significance level. 
To do this use the qf() function.

Test the null hypothesis that the regression $R^2 = 0$.

In [45]:
# Find the critical value of the F-statistic 
# at the 1% significance level

alpha = 0.01

criticalF = qf(p=alpha, df1=1, df2=418, lower.tail = FALSE) 
#criticalF = qf(p=alpha, df1=1, df2=Inf, lower.tail = FALSE) 
criticalF

[1] 6.634897

#### Enter your answer here 

$ F_{1,418} = 6.70$ 

Hence you can reject the null hypothesis of no linear relationship between test scores and the percent of students eligible for subsidized lunch. Alternatively, you can also use $F_{1,\infty}= 6.63$. Qualitatively, the results and the conclusion are the same. 

In [6]:
## DO NOT EDIT
# This section formats your answers for marking
answers = c(t.relprod, z.2sided.1, z.2sided.5, alpha, criticalF)

marksheet <- c(-5.5, -2.5758293035489, -1.95996398454005, 0.01, 6.69589168904183)

round(answers,3) == round(marksheet,3)

[1] TRUE TRUE TRUE TRUE TRUE